llvm/lib/ProfileData/InstrProf.cpp

```diff
@@ -1650,10 +1659,12 @@ size_t Header::size() const {
     // When a new field is added to the header add a case statement here to
     // compute the size as offset of the new field + size of the new field. This
     // relies on the field being added to the end of the list.
-    static_assert(IndexedInstrProf::ProfVersion::CurrentVersion == Version12,
+    static_assert(IndexedInstrProf::ProfVersion::CurrentVersion == Version13,
                   "Please update the size computation below if a new field has "
                   "been added to the header, if not add a case statement to "
                   "fall through to the latest version.");
+  case 13ull:
```

Comment:
Hmm, I just realized this works for version 13 but doesn't initialize 'Header::size'. Working on a fix now.

Reply:
done in this [commit](https://github.com/llvm/llvm-project/pull/88212/commits/3ac8075b1f893e91f4a3bd5495f4494c87b752eb).